lookout/style/format/analyzer.py

```diff
@@ -70,6 +71,7 @@ class FormatAnalyzer(Analyzer):
             "n_iter": 5,
             "line_length_limit": 500,
             "lower_bound_instances": 500,
+            "cutoff_label_precision": 0.95,
```

Comment:
Should probably be grouped with cutoff_label_support

Reply:
So support filter is run in FeatureExtractor and precision filter in `train()`. I had them grouped in the beginning but that made the code complex.